docs/core-api/FILES.md

```diff
@@ -475,20 +476,24 @@ Here, each `path` corresponds to the name of a file, and `content` is an async i
 const cid = 'QmQ2r6iMNpky5f1m4cnm3Yqw8VSvjuKpTcK1X7dBR1LkJF'
 
 for await (const file of ipfs.get(cid)) {
-  console.log(file.path)
+  console.log(file.type, file.path)
 
   if (!file.content) continue;
 
   const content = []
 
-for await (const chunk of file.content) {
+  for await (const chunk of file.content) {
     content.push(chunk)
   }
 
   console.log(content)
 }
 ```
 
+When invoking this method via the HTTP API client, the response arrives as a stream in the tar format containing the entire contents of the file or directory tree.
```

Comment:
the user doesnt care about stream being a tar, maybe its best to not mention tar here to avoid confusion

Reply:
Good shout, I've updated the text